src/watcher.ts

```diff
@@ -1,39 +1,61 @@
 import chokidar from 'chokidar';
 import { exec, ChildProcess } from 'child_process';
+import type Serverless from 'serverless';
 
-export type Config = {
+type ConfigItem = {
   path: string | string[];
-  command: string;
-}[];
+  command?: string;
+  hooks?: string | string[];
```

Comment:
For API consistency, let's call this the singular `hook`.

If I was redoing this now I'd probably just call them `commands` and `hooks`, but I think it's not worth the effort to have to get people to migrate / deprecate old ones etc.